tools/deb/istio-iptables.sh

```diff
@@ -104,8 +104,7 @@ if [ -z "${PROXY_UID}" ]; then
   # Default to the UID of ENVOY_USER and root
   PROXY_UID=$(id -u ${ENVOY_USER:-istio-proxy})
   if [ $? -ne 0 ]; then
-     echo "Invalid istio user $PROXY_UID $ENVOY_USER"
-     exit 1
+     PROXY_UID="1337"
```

Comment:
This will likely still fail if the 1337 user does not exist. see https://github.com/istio/istio/issues/4500

Reply:
You mean in some cases setting a UID of a non-existing user will fail ? 

This is what we had in the original code, this is a regression - we can improve it in separate PR.